### PR TITLE
Correct command to create new plugins in create-app readme

### PR DESCRIPTION
**Pull request by afscrome**
## Hey, I just made a Pull Request!

Correct command to create a new plugin in `plugins/readme.md` created by create-app.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/create-app/templates/default-app/plugins/README.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ This is where your own plugins and their associated modules live, each in a
 separate folder of its own.
 
 If you want to create a new plugin here, go to your project root directory, run
-the command `yarn backstage-cli create`, and follow the on-screen instructions.
+the command `yarn new`, and follow the on-screen instructions.
```
Comment (clanghout) [inline packages/create-app/templates/default-app/plugins/README.md:7]: Should it be `yarn new --select plugin` to be more explicit? I don't know if they are different or not. 

Optionally you can also add a link to [the docs](https://backstage.io/docs/plugins/create-a-plugin/)?

Comment (afscrome) [reply packages/create-app/templates/default-app/plugins/README.md:7]: `yarn new --select plugin` is specifically for a front end plugin.  `yarn new` gives you an option to pick what type of plugin you want

image

Comment (clanghout) [reply packages/create-app/templates/default-app/plugins/README.md:7]: Oooh never knew that and always went to the docs for the specific frontend and backend commands 🤦 all the more reason your changes help!

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Nice thanks!

**Pull request merged**